h/static/scripts/controllers.coffee

```diff
@@ -22,13 +22,13 @@ class AppController
   this.$inject = [
     '$controller', '$document', '$location', '$route', '$scope', '$window',
     'auth', 'drafts', 'identity',
-    'permissions', 'streamer', 'annotationUI',
+    'permissions', 'streamer', 'annotationUI', 'host',
```

Comment:
True, no public API of the host service is called (in this version), but simply instantiating this host enables in to function in the background. So it's not strictly unused. 

(Anyway, the controller will start using the host service in the direct linking work; that's what I put it here.) Can you suggest a different location to place this "unused" import?